xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.cpp

```diff
@@ -300,6 +307,44 @@ bool CProcessorHD::CheckFormats() const
   return IsFormatSupported(DX::Windowing()->GetBackBuffer().GetFormat(), D3D11_VIDEO_PROCESSOR_FORMAT_SUPPORT_OUTPUT);
 }
 
+bool CProcessorHD::IsFormatConversionSupported(DXGI_FORMAT inputFormat,
+                                               DXGI_FORMAT outputFormat,
+                                               const VideoPicture& picture) const
+{
+  // accept the conversion unless the API can be called successfully and disallows it
+  BOOL supported{TRUE};
+  ComPtr<ID3D11VideoProcessorEnumerator1> enumerator1{};
```

Comment:
```suggestion
  ComPtr<ID3D11VideoProcessorEnumerator1> enumerator1;
```
not need initialize `ComPtr` same as `std::string`, etc.